carlotEvents.js

```diff
@@ -0,0 +1,44 @@
+// function activateEvents() {
+
+var carLot = (function(eventHandlers) {
+
+
+
+    var input = document.getElementById("descriptionInput");
+    var currentCar;
+
+    eventHandlers.addHandlers = function(car, inventory) {
+
+
+
+        //triggers the click effect of each specific card
+        car.addEventListener("click", function() {
+            currentCar = event.currentTarget;
+            var allCars = document.getElementsByClassName("card");
+            for (var i = 0; i < allCars.length; i++) {
+                let carCards = allCars[i]
+                carCards.style.borderWidth = "1px";
+                carCards.style.backgroundColor = "cornsilk";
+            }
+            // currentCar = event.currentTarget;
+            console.log("currentCar", currentCar);
+            console.log("currentCar", currentCar.querySelector(".carInfo"));
+            eventHandlers.clickEffect(currentCar, inventory, input);
+
+
+        });
+        input.addEventListener("keyup", function() {
+            var outPutTarget = currentCar.children[2];
+            outPutTarget.innerHTML = input.value;
+
+        });
+
```

Comment:
You've got this working... but technically you're adding this event 6 times. (one for each car in your JSON). You can tell this because it also fires console errors if you type anything in the box before clicking a card. I would suggest fixing the errors, and in the process you might try only attaching the eventlistener to the input box once.